app/views/users/index.html.erb

```diff
@@ -1,2 +1,25 @@
-<h1>List of users</h1>
-<p>Users get listed here!</p>
\ No newline at end of file
+<h1 class = "p-4 text-center " >List of all users</h1>
+
+<% @users.each do |user| %>
+
+<div class="d-flex row justify-content-center">
+  <div class= "card text-dark bg-light  p-0 m-2" style="max-width: 10rem">
+
+      <div class=" border-end pr-2">
+          <%= image_tag("images.png", :alt => "rss feed", class: 'img-fluid rounded-start')   %>
+    </div>
+  </div>
+
+  <div class="card text-dark bg-light m-2 d-flex flex-row" style="max-width: 600px;">
+    <div class="col-md-4  d-flex align-items-center">
+      <h5 class="card-title"> <%= link_to user.name, users_url(user), class: 'display-5 text-primary' %></h5> 
+    </div>
+
+    <div class = " col-md-7 d-flex align-items-end justify-content-end" >
+      <p class="text-right">Number of posts: 
+        <%= user.posts_counter ?  user.posts_counter : 0   %>
+        </p>
+      </p>
```

Comment:
Currently, your app is not according to the design provided -
- [ ] Kindly make sure to follow the wireframe as [given](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md).
- [ ] Make sure that when the `User name` is clicked all posts are displayed. 
- [ ] Make sure that When `see all posts` button is clicked all posts with comments are shown.
- [ ] Make sure that when one post is clicked all the comments of that specific post are shown.

____________________________________________

- Kindly look below when I clicked user name I am not able to view the required page.

https://user-images.githubusercontent.com/70269507/173761628-20352e46-da4c-41d5-a711-3c5ff6b94211.mov

- As you can see below I have tried adding users, posts and comments but still it's not displayed.

<img width="438" alt="Screenshot 2022-06-15 at 9 28 12 AM" src="https://user-images.githubusercontent.com/70269507/173761638-5bf3c741-dda8-45bf-b298-556e2933ab85.png">

Reply:
Hello @sja-thedude Thanks for suggestion I fix the issues
![sample-1](https://user-images.githubusercontent.com/93598189/173839971-0a06cf51-3ccf-49d3-90b6-a1f7e6bdd92c.gif).
![sample-2](https://user-images.githubusercontent.com/93598189/173840032-c9321e6b-931e-4646-b470-de90c92e4ffa.gif)
![sample-3](https://user-images.githubusercontent.com/93598189/173840066-98d5a1d8-f2d7-4dcc-8d3b-993c4295e347.gif)